turborepo-tests/integration/fixtures/root_deps/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "monorepo",
+  "workspaces": [
+    "apps/**",
+    "packages/**"
+  ],
+  "dependencies": {
+    "util": "*"
+  }
+}
```

Comment:
I can't remember if our integration test fixtures generally use `packageManager` or not, but that might be useful since I know `*` has some weird behavior in different npm versions, especially with `workspaces`.

Reply:
> I can't remember if our integration test fixtures generally use packageManager or not, but that might be useful since I know * has some weird behavior in different npm versions, especially with workspaces

Good callout. With package manager requirements we now force usage of corepack for all test fixture defaulting to `npm` if one isn't specified in the fixture. Will open a ticket to make this explicit